plugins/catalog/api-report.md

```diff
@@ -330,6 +330,14 @@ export interface DependsOnResourcesCardProps {
 // @public (undocumented)
 export const EntityAboutCard: (props: AboutCardProps) => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "EntityContextMenuProps" needs to be exported by the entry point index.d.ts
+// Warning: (ae-missing-release-tag) "EntityContextMenu" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Note these two warnings. Especially the one about types. One of the reasons that this one wasn't exported so far, is that it has a pretty complex set of type dependencies. When you import this component you won't be getting proper types for its props. We also would probably like to NOT export all of those types because it expands the footprint of this package with noise.

Let's merge for now, but if this proves problematic, it would be preferable to _inline_ the types used in the props by hand, eg rewriting the menu as 

```ts
export function EntityContextMenu(props: {
  UNSTABLE_extraContextMenuItems?: Array<{
    title: string;
    Icon: IconComponent;
    onClick: () => void;
  }>;
  UNSTABLE_contextMenuOptions?: {
    disableUnregister: boolean | 'visible' | 'hidden' | 'disable';
  };
  onUnregisterEntity: () => void;
  onInspectEntity: () => void;
})
```